client/mysqlimport.c

```diff
@@ -410,14 +410,12 @@ static void lock_table(MYSQL *mysql, int tablecount, char **raw_tablename)
 
 
 
-static MYSQL *db_connect(char *host, char *database,
-                         char *user, char *passwd)
+static void db_connect(char *host, char *database,
```

Comment:
Changed to `const char*`. On your other comment, I actually prefer to pull `mysql_init()` out :). `mysql_init()` and `db_connect()` look to me two separate logic.